lang/en/bigbluebuttonbn.php

```diff
@@ -423,9 +423,8 @@
 $string['view_recording_date'] = 'Date';
 $string['view_recording_description'] = 'Description';
 $string['view_recording_length'] = 'Length';
-$string['view_recording_meeting'] = 'Meeting';
 $string['view_recording_duration'] = 'Duration';
-$string['view_recording_recording'] = 'Recording';
+$string['view_recording_recording'] = 'Name'; // Change Recording column into "Name" as we removed Meeting column - CONTRIB-7703.
```

Comment:
Can you add a new string instead. Re-wordings like this normally cascade issues with translations as the comments are not shown in AMOS.

`view_recording_name`

And don't remove the old ones, this way we keep backward compatibility.

Reply:
Ok. I did not think about AMOS !
There was a language string view_recording_name which value was Name, so I used it.